app/controllers/v1/people_controller.rb

```diff
@@ -1,7 +1,7 @@
 class V1::PeopleController < V1::BaseController
 
   def create
-    @person = PersonWithRemoteFields.find_or_build(person_params)
+    @person = PersonConstructor.new(person_params).build
```

Comment:
I think PersonConstructor.new should return an instance of PersonConstructor. It's counter-intuitive to have it return anything else.

Theoretically, PersonConstructor will also have a `create` method and maybe an `attributes` method at some point. I'm trying to avoid `create` for now, though, because I think overuse of create (i.e., `Person.create_or_update`) has resulted in none of our controllers actually verifying that the person was created successfully.